io.github.gameflorist.dunedynasty.metainfo.xml

```diff
@@ -0,0 +1,81 @@
+  <launchable type="desktop-id">io.github.gameflorist.dunedynasty.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>Dune Dynasty</name>
+  <summary>Remaster/Enhancement of the classic real-time strategy game Dune II by Westwood Studios</summary>
+  <description>
+    <p>The goal of <em>Dune Dynasty</em> is is to make Dune II playable on modern systems with additional features, bugfixes and QoL-improvements.</p>
+    <p>It is not a remake. It builds upon the original game engine as reverse-engineered by the OpenDUNE project.</p>
+    <p>NOTE: Original Dune II game files are required.</p>
+    <p>Improvements include:</p>
+    <ul>
+      <li>Support for modern operating systems (Windows, Linux, MacOS)</li>
+      <li>High-resolution widescreen graphics &amp; customizable scaling of for HiDPI displays</li>
+      <li>Control Enhancements like scrollable sidepanel, building queue, selecting/commanding multiple units, control groups,etc.</li>
+      <li>Skirmish &amp; Multiplayer for up to 6 players/AI with custom teams</li>
+      <li>Music Enhancements: Emulated AdLib, MIDI (via system or FluidSynth), external music sets (e.g. for Sega Mega Drive / Amiga music, other Dune games or movies)</li>
+      <li>Gameplay Enhancements: Health bar, Fog of war, raised structure and unit limits, start level selection and many more.</li>
+      <li>Support for various custom campaigns made by the community</li>
+      <li>Various bug fixes</li>
+    </ul>
+  </description>
+  <url type="homepage">https://github.com/gameflorist/dunedynasty</url>
+  <url type="bugtracker">https://github.com/gameflorist/dunedynasty/issues</url>
+  <developer id="io.github.gameflorist.dunedynasty">
+    <name>Dune Dynasty contributors</name>
+  </developer>
+  <recommends>
+    <control>keyboard</control>
+    <control>pointing</control>
+  </recommends>
+  <requires>
+    <display_length compare="ge">360</display_length>
+  </requires>
+  <categories>
+    <category>Game</category>
+    <category>2DGraphics</category>
+    <category>StrategyGame</category>
+  </categories>
+  <keywords>
+    <keyword>IDE</keyword>
+    <keyword>development</keyword>
+  </keywords>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-realistic">moderate</content_attribute>
+    <content_attribute id="violence-bloodshed">moderate</content_attribute>
+    <content_attribute id="violence-desecration">mild</content_attribute>
+    <content_attribute id="drugs-narcotics">mild</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+  </content_rating>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/gameflorist/dunedynasty/refs/heads/v164/docs/screenshot-1.jpg</image>
```

Comment:
this should NEVER reference the tip of a branch. Use a specific commit or tag.

(same below)